javaparser-core-testing/src/test/java/com/github/javaparser/modules/ModuleDeclarationTest.java

```diff
@@ -178,4 +181,24 @@ public void testCsmPrinting() {
                         "}" + EOL, ConcreteSyntaxModel.genericPrettyPrint(cu));
 
     }
+
+    @Test
+    public void fluentInterface() {
+        ModuleDeclaration moduleDeclaration = new CompilationUnit()
+                .addModule("org.javacord.api")
+                .addSingleMemberAnnotation(Generated.class, "generator")
+                .addSingleMemberAnnotation(SuppressWarnings.class, "\"module\"")
+                .addDirective("requires transitive java.desktop;")
+                .addDirective("exports org.javacord.api.entity.channel;")
+                .addDirective("exports org.javacord.api.entity.channel.internal to org.javacord.core;")
+                .addDirective("uses org.javacord.api.util.internal.DelegateFactoryDelegate;");
+
+        assertEqualsNoEol("@Generated(generator) @SuppressWarnings(\"module\") \n" +
+                "module org.javacord.api {\n" +
+                "    requires transitive java.desktop;\n" +
+                "    exports org.javacord.api.entity.channel;\n" +
+                "    exports org.javacord.api.entity.channel.internal to org.javacord.core;\n" +
+                "    uses org.javacord.api.util.internal.DelegateFactoryDelegate;\n" +
+                "}\n", moduleDeclaration.toString());
+    }
```

Comment:
Would you mind replacing by dummy names?
I'm not sure, it being a good practice to contain other projects code in your tests.
And remove the `Generated`, unfortunately the compiler complains that Generated (neither the old, nor the new one) target modules.

Reply:
Oh sure!